static/app/views/performance/browser/webVitals/pageOverviewWebVitalsDetailPanel.tsx

```diff
@@ -304,17 +347,31 @@ export function PageOverviewWebVitalsDetailPanel({
           {webVital && <WebVitalStatusLineChart webVitalSeries={webVitalData} />}
         </ChartContainer>
         <TableContainer>
-          <GridEditable
-            data={tableData}
-            isLoading={isTransactionWebVitalsQueryLoading}
-            columnOrder={columnOrder}
-            columnSortBy={[sort]}
-            grid={{
-              renderHeadCell,
-              renderBodyCell,
-            }}
-            location={location}
-          />
+          {isInp ? (
+            <GridEditable
+              data={inpTableData as unknown as InteractionSpanSampleRowWithScore[]}
```

Comment:
Not sure if you added this before typing the hook, but it doesn't seem necessary anymore.

Reply:
👍 cleaned up!